jetty-integrations/jetty-nosql/src/main/java/org/eclipse/jetty/nosql/mongodb/MongoSessionDataStore.java

```diff
@@ -175,6 +175,22 @@ public MongoCollection<Document> getDBCollection()
         return _dbSessions;
     }
 
+    @Override
+    protected void doStart() throws Exception
+    {
+        checkWorkerName();
+        super.doStart();
+    }
+
+    private void checkWorkerName() throws IllegalStateException
+    {
+        if (_context == null || StringUtil.isEmpty(_context.getWorkerName()))
+            return;
+
+        if (!_workerNamePattern.matcher(_context.getWorkerName()).matches())
+            throw new IllegalStateException("Invalid worker name: " + _context.getWorkerName());
```

Comment:
Works like a charm!
Only improvement might be stating what's valid, so if the issue is encountered it'd be fixable without diving into the code and looking up the Pattern